actions/class.Authoring.php

```diff
@@ -43,7 +46,7 @@ public function index()
     	$testService = taoTests_models_classes_TestsService::singleton();
 
     	$class = new core_kernel_classes_Class(CLASS_LTI_TESTCONTENT);
-    	$content = $test->getOnePropertyValue(new core_kernel_classes_Property(TEST_TESTCONTENT_PROP));
+    	$content = $test->getOnePropertyValue(new core_kernel_classes_Property(TestService::PROPERTY_TEST_CONTENT));
```

Comment:
depends on https://github.com/oat-sa/extension-tao-test/pull/188/files#diff-096f34b8841df0fda7fda714e51755f8R37 (v6.5.1)

Reply:
@ionutpad I'm waiting for update

Reply:
Done, Thanks